### ISSUE TITLE
Kolbot grabbing items that I have marked off in my pickit file to not pick up!

### ISSUE THREAD

**Issue by IamDino225** [kolton/d2bot-with-kolbot#2618]
Sorry if this issue has been solved, I looked around and couldn't find it.

![image](https://user-images.githubusercontent.com/67375033/85562986-997c6180-b5f2-11ea-9b71-a8f47ea7bf9a.png)

its saying ring  in line #154 leading me to believe I farmed an soj while I was gone, but sadly, it was a dwarf star which I have turned off currently

![image](https://user-images.githubusercontent.com/67375033/85563219-cfb9e100-b5f2-11ea-9338-24ddd65ecc55.png)

**Comment by DarkHorseDre** [comment]
how many pickit files are you using? (you can confirm this at the start of the run, press home key ingame to see console output - nip files will be listed at the top)

**Comment by IamDino225** [comment]
Just one!

**Comment by DarkHorseDre** [comment]
yeah thats what they all say ;)

ok I assume you're reloaded/restarted the bot since making the last nip change?

if so, please paste the chars config to pastebin.com (or similar) and paste the link here

Also do all the other items it has found since then show a line number out by one?

(also note your amu and ring are unid but would be id'd if that nip file was used..)

**Comment by 5noop** [comment]
When you have `[affix] <= x` it means the affix can also be 0 which is valid for any item, this is why the ring was kept unid.

**Comment by DarkHorseDre** [comment]
> When you have `[affix] <= x` it means the affix can also be 0 which is valid for any item, this is why the ring was kept unid.

damn I didnt even look - assumed he used the original line 🤦

**Comment by IamDino225** [comment]
> yeah thats what they all say ;)
> 
> ok I assume you're reloaded/restarted the bot since making the last nip change?
> 
> if so, please paste the chars config to pastebin.com (or similar) and paste the link here
> 
> Also do all the other items it has found since then show a line number out by one?
> 
> (also note your amu and ring are unid but would be id'd if that nip file was used..)

I promise I'm only doing one lol, and yes I have!
 I will copy and paste for ya when get back home

**Comment by IamDino225** [comment]
> When you have `[affix] <= x` it means the affix can also be 0 which is valid for any item, this is why the ring was kept unid.

Does this mean I should change it from < to >
And that would make the bot Id it?

**Comment by IamDino225** [comment]
https://pastebin.com/Udpz1ZtD

**Comment by DarkHorseDre** [comment]
Compare the unique ring lines in your pickit with the default kolton.nip file (if you overwrote it, you can find it here: https://github.com/blizzhackers/kolbot/tree/master/d2bs/kolbot/pickit

**Comment by IamDino225** [comment]
> Compare the unique ring lines in your pickit with the default kolton.nip file (if you overwrote it, you can find it here: https://github.com/blizzhackers/kolbot/tree/master/d2bs/kolbot/pickit

okay ill just mess with that one

**Comment by IamDino225** [comment]
So after un/reinstalling a couple times and using different pickit scripts, I am still having the same issue.
Anyone else ever encounter this?

**Comment by oledadz** [comment]
yes i have the same problem when i turn off stuff it still picks it up like its ignoring it.
ive tried removing certain rings and ammys, gems amd he still picks them up. didnt even have chipped gems turned on he picked them up so i tried different pickits, restarting comp every thing i never had this problem with the old bots like kukbot,etal it dont make no since.
